plugins/filesystem/local/adapter/adapter.php

```diff
@@ -326,4 +326,272 @@ private function getDate($date = null)
 
 		return $dateObj;
 	}
+
+	/**
+	 * Copies a file or folder to a destination
+	 * If the destination folder or file already exists, it will not overwrite them without
+	 * force.
+	 *
+	 * @param   string  $sourcePath       Source path of the file or directory
+	 * @param   string  $destinationPath  Destination path of the file or directory
+	 * @param   bool    $force            Set true to overwrite files or directories
+	 *
+	 * @return bool
+	 *
+	 * @since __DEPLOY_VERSION__
+	 * @throws MediaFileAdapterFilenotfoundexception
+	 */
+	public function copy($sourcePath, $destinationPath, $force = false)
```

Comment:
If you default force to false here, then you need to do that in the interface as well.